packages/webapp/pages/sources/[source].tsx

```diff
@@ -168,6 +168,15 @@ export async function getStaticProps({
       id: params.source,
     });
 
+    if (res.source?.type === 'squad') {
+      return {
+        redirect: {
+          destination: `/squads/${params.source}`,
```

Comment:
Looking at another pass we might want to reapply any query params since this does not do it automatically as redirect in `next.config.js`.

Reply:
Very good point!
For now it should be ok as staticProps doesn't have access to them, we should move to getServerSideProps as discussed.

I think we should push now, and add a ticket to refactor it slowly.

Note: we don't really advertise these with UTM's much so shouldn't block much I don't think.
@idoshamun agreed?

Reply:
I'm cool with that 😀